packages/just-task/package.json

```diff
@@ -28,10 +28,13 @@
     "resolve": "^1.19.0",
     "undertaker": "^1.3.0",
     "undertaker-registry": "^1.0.1",
-    "yargs-parser": "^20.2.3"
+    "yargs-parser": "^20.2.3",
```

Comment:
Shouldn't `chokidar` be in the dependencies here?

Reply:
chokidar is a dep in glob-watcher not here.

Reply:
I'm using @types/chokidar, not chokidar. Basically only importing the watcher types.

Reply:
chokidar itself is a dep of glob-watcher...

Reply:
oh got it, i see now.